pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go

```diff
@@ -19,9 +27,29 @@ type MeshDefaultsReconciler struct {
 }
 
 func (r *MeshDefaultsReconciler) Reconcile(req kube_ctrl.Request) (kube_ctrl.Result, error) {
+	mesh := core_mesh.NewMeshResource()
+	if err := r.ResourceManager.Get(context.Background(), mesh, store.GetByKey(req.Name, core_model.NoMesh)); err != nil {
+		return kube_ctrl.Result{}, errors.Wrap(err, "could not get default mesh resources")
+	}
```

Comment:
I'm not sure that we should re-queue here. Normally, the error on get means the object was deleted, though in this case we are only looking for `add` events, so we wouldn't expect that. At any rate, I don't think that there's any reason to believe that re-queuing the event would help.

Reply:
I think it makes sense to distinguish errors here, if it's a connectivity problem, then I think retry should happen. So I added the following check:
```go
if kube_apierrs.IsNotFound(err) {
	return kube_ctrl.Result{}, nil
}
```
WDYT?